src/trie.c

```diff
@@ -46,8 +46,6 @@ int trie_free(trie_t *t)
     return 0;
 }
 
-
-/* See trie.h */
 int add_node(char current, trie_t *t)
```

Comment:
As discussed in the early lectures, every function that's part of a "module" (even though C doesn't have real modules) should have the same prefix. This should be `trie_add_node`. Further, the `trie_t *` should always be the first argument.

Reply:
See above